test/integration/channels.js

```diff
@@ -123,7 +123,7 @@ describe('Channels - Constructor', function () {
 
 	it('short version with options', function (done) {
 		const channel = client.channel('messaging', { members: ['tommaso', 'thierry'] });
-		expect(channel.data).to.eql({ members: ['tommaso', 'thierry'] });
+		expect(channel.data).to.eql({ members: ['thierry', 'tommaso'] });
```

Comment:
this test seems to be a better fit for unit tests. is this due to recent changes? the member order should remain the same

Reply:
yes, it comes from member based channel creation fix.

I agree with unit.

Reply:
oh I see, the sort is applied in place, I think it's better to revert that change

Reply:
I will make a fix to sort the copy and move this test to unit tests.

Reply:
@ferhatelmas https://github.com/GetStream/stream-chat-js/pull/592

Reply:
ahh, updated this one